updated how-to guides (#63)
dappforce/subsocial-docs: docs/develop/how-to-guides/comments/FetchComments.md

---
**This section covers how to fetch comments on the Subsocial blockchain.**

> Comments in Subsocial Blockchain are a type of post and as we call it Extension. You can read more about it [here](/docs/develop/how-to-guides/posts/create-posts). 

Reviewer: ```suggestion
Comments on the Subsocial blockchain are technically a type of post classified as Extensions. You can read more about it [here](/docs/develop/how-to-guides/posts/create-posts). 
```

Not entirely sure about this suggestion, does it look good to you?
